test/liquid/combinators/tags/assign_test.exs

```diff
@@ -138,11 +126,11 @@ defmodule Liquid.Combinators.Tags.AssignTest do
     )
   end
 
-  test "incorrect variable assignment" do
-    test_combinator_error("{% assign cart@ = 5 %}", &Parser.assign/1)
-    test_combinator_error("{% assign cart. = 5 %}", &Parser.assign/1)
-    test_combinator_error("{% assign .cart = 5 %}", &Parser.assign/1)
-    # TODO: this must fails
-    # test_combinator_error("{% assign cart? = 5 %}", &Parser.assign/1)
-  end
+  # test "incorrect variable assignment" do
```

Comment:
Why is this commented out? either add `@skip true` to skip the test or remove it all together. This does look like a case we should test.

Reply:
Done. Test was uncommented.